minitest/repositores/company_repository.py

```
@@ -0,0 +1,21 @@
+    def get_company(self, idx=None):
+        return tmp_company_db
+    
+    def create_company(self, company):    
+        tmp_company_db.append(company)    
+        return tmp_company_db
+    
+    def find_by_name(self, name: str):
+        for company in tmp_company_db:
+            if company.name==name:
+                return True
+        return False
```

Ruff (0.8.2)

Use `return any(company.name == name for company in tmp_company_db)` instead of `for` loop

Replace with `return any(company.name == name for company in tmp_company_db)`

(SIM110)